### PR TITLE
Try adding arm64 (DEV-79)

### PR DESCRIPTION
**Pull request by alexeyzimarev**
I just took some code from the terraform provider repo (arm64 for goreleaser) and replicator (Docker)

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -11,44 +11,68 @@ jobs:
...
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout
-        uses: actions/checkout@v2
-        with:
-          fetch-depth: 0
-      - name: Set up Go
+      -
+        name: Checkout
+        uses: actions/checkout@v2.3.4
```
Comment (thefringeninja) [inline .github/workflows/ci.yml]: Actions which rely on Node 12 are deprecated. For any of the GitHub made ones we can safely upgrade to @v3.

> Node.js 12 actions are deprecated. For more information see: https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/. Please update the following actions to use Node.js 16: actions/checkout@v2.3.4, actions/setup-go@v2, docker/metadata-action@v3, docker/setup-qemu-action@v1, docker/setup-buildx-action@v1, docker/build-push-action@v2

**Review by thefringeninja — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (alexeyzimarev) [reply .github/workflows/ci.yml]: All done

Comment (thefringeninja) [reply .github/workflows/ci.yml]: Still getting a warning for using @v2 of the checkout action: https://github.com/EventStore/es-gencert-cli/actions/runs/3547339752

Comment (alexeyzimarev) [reply .github/workflows/ci.yml]: Yeah, but 2.5.0 is the latest release https://github.com/actions/checkout/releases

Comment (alexeyzimarev) [reply .github/workflows/ci.yml]: Ok, I see that 3.1 is there.

Comment (alexeyzimarev) [reply .github/workflows/ci.yml]: No more warnings

**Review by thefringeninja — APPROVED** [review]

**Pull request merged**